src/v2/styles/AttachmentList/AttachmentList-layout.scss

```diff
@@ -415,10 +414,6 @@
       width: calc(var(--str-chat__spacing-px) * 45);
     }
 
-    stream-voice-recording-wavebar {
-      flex: 1;
-    }
```

Comment:
Angular need this, please don't remove it

Reply:
@szuperaz  I have reverted the change and documented the selector